Emby.Server.Implementations/Activity/ActivityLogEntryPoint.cs

```diff
@@ -76,7 +72,6 @@ public Task RunAsync()
             _sessionManager.PlaybackStart += _sessionManager_PlaybackStart;
             _sessionManager.PlaybackStopped += _sessionManager_PlaybackStopped;
 
-            //_subManager.SubtitlesDownloaded += _subManager_SubtitlesDownloaded;
```

Comment:
I think we should re-enable it instead

Reply:
The event got removed as nothing ever triggers it.

Reply:
Was it there for plugins to use by any chance?

Reply:
And yes, I agree here that keeping it might be worth it for activity DB and some reports, maybe plugins, too.

Reply:
The event doesn't even exist anymore, we cant keep it in.